frontend/src/config/afghanistan/prism.json

```diff
@@ -13,6 +13,7 @@
   "serversUrls": {
     "wms": ["https://api.earthobservation.vam.wfp.org/ows/wms"]
   },
+  "chartApiUrl": "https://prism-api.ovio.org/hdc",
```

Comment:
If this is going to be the same for every country, this could even live in a constants.ts, similarly to what's done in the PR linked above